FCCee/compact/FCCee_o1_v04/FCCee_o1_v04.xml

```diff
@@ -298,6 +298,59 @@
     <plugins>
       <plugin name="DD4hepVolumeManager"/>
       <plugin name="InstallSurfaceManager"/>
+      <plugin name="lcgeo_LinearSortingPolicy">
+
+        <argument value="/InnerTrackerEndcapSupport_layer8" />
+        <argument value="InnerTracker_Barrel_half_length_0" />
+        <argument value="InnerTracker_Barrel_radius_0+0.5*mm" />
+        <argument value="0" />
+
+        <argument value="/InnerTrackerEndcapSupport_layer9" />
+        <argument value="InnerTracker_Barrel_half_length_0" />
+        <argument value="InnerTracker_Barrel_radius_1+0.5*mm" />
+        <argument value="0" />
+
+        <argument value="/InnerTrackerEndcapSupport" />
+        <argument value="InnerTracker_Barrel_half_length_0" />
+        <argument value="InnerTracker_Barrel_radius_1+0.5*mm" />
+        <argument value="(InnerTracker_outer_radius-InnerTracker_Barrel_radius_1)/(InnerTracker_half_length-InnerTracker_Barrel_half_length_0)" />
+
+        <argument value="/InnerTrackerEndcap/" />
+        <argument value="InnerTracker_Barrel_half_length_0" />
+        <argument value="InnerTracker_Barrel_radius_1" />
+        <argument value="(InnerTracker_outer_radius-InnerTracker_Barrel_radius_1)/(InnerTracker_half_length-InnerTracker_Barrel_half_length_0)" />
+
+        <argument value="/OuterTrackerEndcap/" />
+        <argument value="OuterTracker_Barrel_half_length" />
+        <argument value="OuterTracker_Barrel_radius_1" />
+        <argument value="(OuterTracker_Endcap_radius_2-OuterTracker_Barrel_radius_1)/(OuterTracker_half_length-OuterTracker_Barrel_half_length)" />
+
+        <argument value="/OuterTrackerEndcapSupport_layer4" />
+        <argument value="OuterTracker_Barrel_half_length" />
+        <argument value="OuterTracker_Barrel_radius_0+0.5*cm" />
+        <argument value="0.0" />
+
+        <argument value="/OuterTrackerEndcapSupport_layer5" />
+        <argument value="OuterTracker_Barrel_half_length" />
+        <argument value="OuterTracker_Barrel_radius_1+0.5*cm" />
+        <argument value="0.0" />
+
+        <argument value="/OuterTrackerEndcapSupport" />
+        <argument value="OuterTracker_Barrel_half_length" />
+        <argument value="OuterTracker_Barrel_radius_1+3*cm" />
+        <argument value="(OuterTracker_Endcap_radius_2-OuterTracker_Barrel_radius_1)/(OuterTracker_half_length-OuterTracker_Barrel_half_length)" />
+
+        <argument value="/VertexEndcap/" />
+        <argument value="VertexEndcap_zmin" />
+        <argument value="VertexBarrel_r3+1*cm" />
+        <argument value="(VertexEndcap_rmax-VertexBarrel_r3+1*cm)/(VertexEndcap_zmax-VertexEndcap_zmin)" />
```

Comment:
VertexEmdcap_zmin and VertexEmdcap_zmax are not defined

Reply:
replaced with VertexEndcap_z1 and VertexEndcap_z3 respectively